sbol/test/test_document.py

```diff
@@ -159,5 +161,17 @@ def test_find_property_value(self):
         matches = doc.find_property_value(sbol.SBOL_IDENTITY, test_uri)
         self.assertEqual(len(matches), 0)
 
+    def test_add_namespace(self):
+        doc = sbol.Document()
+        cd = doc.componentDefinitions.create('cd')
+        cd.foo = sbol.property.LiteralProperty(cd, 'http://examples.org#', '0', '1', 'bar')
+        doc.readString(doc.writeString())
+        namespaces = [n[1] for n in doc.graph.namespace_manager.namespaces()]
+        self.assertFalse('http://examples.org#' in namespaces)
+        doc.addNamespace('http://examples.org#', 'foo')
+        doc.readString(doc.writeString())
+        namespaces = [n for n in doc.graph.namespace_manager.namespaces()]
+        self.assertTrue('foo', URIRef('http://examples.org#') in namespaces)
```

Comment:
I find this visually confusing. Could you add parens around the tuple to make it more clear that you are creating a tuple?

Reply:
Fixed in #f8a7cb3